app/src/main/res/values-en/strings.xml

```diff
@@ -1236,4 +1236,154 @@
+    <string name="update_available_fallback">You have an outdated version of the Szkolny.eu application. You need to update the app to continue to sync data.</string>
+    <string name="update_available_format">You are using an old version of the Szkolny.eu application (%s). To use the app and ensure the best performance, please upgrade to %s. Change log: %s</string>
+    <string name="profile_archived_text">You are viewing a student\'s data from the previous school year (%d /%d). Syncing and downloading of messages and some homework have been disabled. To open a student\'s profile for the current year, select Close Archive on the home page.</string>
+    <string name="error_1201_reason">No parameters were given</string>
+    <string name="error_940_reason">Report bug: exception in PPE API</string>
+    <string name="error_920_reason">An error occured</string>
+    <string name="error_911_reason">Report bug: downloading LIBRUS® messages</string>
+    <string name="error_632_reason">No data. Report bug to developer.</string>
+    <string name="error_601_reason">Invalid token</string>
+    <string name="error_511_reason">No session ID</string>
+    <string name="error_522_reason">Session expired</string>
+    <string name="error_501_reason">Invalid email or password</string>
+    <string name="error_510_reason">Other login error</string>
+    <string name="error_921_reason">An error occured while file downloading</string>
+    <string name="login_mode_librus_jst_guide">Log in to LIBRUS® Synergia on your computer, select the Mobile Applications tab, then enter the received Token and PIN below.</string>
+    <string name="login_mode_librus_synergia_guide">Enter the login received from the school with which you log into LIBRUS® Synergia (purple form).\n\nIt is recommended to log in with the LIBRUS® Family account (using e-mail) in the previous step.</string>
+    <string name="login_mode_librus_email_guide">Log in with your LIBRUS® Rodzina account, which works in the official LIBRUS® application and on the website portal.librus.pl, in the blue form. \n\nIf you do not have a LIBRUS® Rodzina account, you can create one at https://portal.librus.pl/rodzina/register.</string>
+    <string name="login_platform_list_loading_timeout">If it takes too long, please check your internet connection and restart the application.</string>
+    <string name="login_platform_list_subtitle">Select which image corresponds to the one you see when logging in to your log website.  If your school doesn\'t use any of these city platforms, choose the first option.</string>
+    <string name="build_invalid_remote_no_commit">You have an application compilation with unpublished changes. The compilation is in the repository: % 1$s (%2$s) \nwhich is private or does not contain the latest changes.  \n\nFor security reasons and for compliance with the license, the use of the application has been blocked.</string>
```

Comment:
The percent-sign formatting is broken, cannot have the following whitespace. Shouldn't there be a linebreak after `the repository:`? Use `build` instead of `compilation` here and in the lines below.